internal/proxy/helper.go

```diff
@@ -0,0 +1,15 @@
+package proxy
```

Comment:
What about creating a new package called `attribute` inside proxy and do attribute manipulation in it?